manifests/params.pp

```diff
@@ -7,8 +7,20 @@
   $package_version = 'latest'
   $srvadmin = true
   $srvadmin_package = 'srvadmin-all'
+  $srvadmin_packages_java = ['srvadmin-all', 'srvadmin-webserver', 'srvadmin-tomcat', 'srvadmin-jre', ]
+  $srvadmin_packages_no_java = ['srvadmin-argtable2', 'srvadmin-base', 'srvadmin-deng', 'srvadmin-deng-snmp', 'srvadmin-hapi',
```

Comment:
I'm not sure how often this list gets updated, but we should probably look at alternate way

Reply:
Yes your correct. I'm not sure how we can do that without looking at the source of any updated `srvadmin-all` packages. Plus any new versions would require a change. 

What we can do is overwrite the package list with:

`dsu:: srvadmin_packages_no_java = '['package0', 'package1', ...]`

At the moment it's a stop gap to get rid of the java packages.